woocommerce-addressfinder.php

```diff
@@ -3,7 +3,7 @@
 	AddressFinder plugin for autocompleting addresses in WooCommerce for New Zealand and Australia
 	Plugin Name: AddressFinder
```

Comment:
Potentially changing these names to `Addressfinder` will update this section but unsure. 
<img width="694" alt="Screenshot 2023-07-04 at 12 23 47 PM" src="https://github.com/AddressFinder/addressfinder-woocommerce/assets/100241767/b484f25c-0c24-4eca-a0d8-27251c33b863">

Reply:
Yeah, I reckon do it!